### PR TITLE
feat: Fetch community token image

### PR DESCRIPTION
**Pull request by Cuteivist**
Task https://github.com/status-im/status-desktop/issues/12945

corresponding status-desktop change https://github.com/status-im/status-desktop/pull/12980

* Fetched community tokens metadata for custom tokens from app database
* Added token image field for API

### CODE REVIEW

File: services/communitytokens/database.go [hunk 1]
```diff
@@ -45,3 +45,22 @@ func (db *Database) GetTokenPrivilegesLevel(chainID uint64, contractAddress stri
...
+	var result []*token.CommunityToken
+	for rows.Next() {
+		token := token.CommunityToken{}
+		err := rows.Scan(&token.CommunityID, &token.Address, &token.Name, &token.Symbol, &token.ChainID, &token.Base64Image)
+		if err != nil {
+			return nil, err
+		}
+		result = append(result, &token)
+	}
+	return result, nil
```
Comment (saledjenic) [inline services/communitytokens/database.go]:
```suggestion
	return result, rows.Err()
```

File: services/wallet/token/token.go [hunk 2]
```diff
@@ -499,6 +512,12 @@ func (tm *Manager) getTokensFromDB(query string, args ...any) ([]*Token, error)
 
 		if communityIDDB.Valid {
 			token.CommunityID = &communityIDDB.String
+			for _, communityToken := range communityTokens {
+				if communityToken.Symbol == token.Symbol && uint64(communityToken.ChainID) == token.ChainID && communityToken.CommunityID == *token.CommunityID {
```
Comment (saledjenic) [inline services/wallet/token/token.go]: small optimization
```suggestion
				if communityToken.CommunityID != *token.CommunityID || uint64(communityToken.ChainID) != token.ChainID || communityToken.Symbol != token.Symbol {
					continue
				}
```

**Review by saledjenic — APPROVED** [review]

**Review by IvanBelyakoff — APPROVED** [review]
Great job!

File: services/communitytokens/database.go (hunk 1 above)
Comment (Cuteivist) [reply services/communitytokens/database.go]: Good catch. Fixed

**Pull request merged**